source/agora/consensus/data/ConsensusData.d

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+
+    Defines the data used when reaching consensus.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.consensus.data.ConsensusData;
+
+import agora.common.Set;
+import agora.consensus.data.Enrollment;
+import agora.consensus.data.Transaction;
+
+/// Consensus data which is nominated & voted on
+public struct ConsensusData
```

Comment:
Please add a `testSymmetry` check

Reply:
`
/// ConsensusData type testSymmetry check
unittest
{
    import agora.common.Set;
    import agora.common.Types;
    import agora.consensus.data.Transaction;

    struct Data
    {
        public Set!Transaction tx_set;
    }

    testSymmetry!Data();
}

`

Ah, I think I found a bug.

Reply:
What's the output ?

Reply:
If this is too much to workaround, it can be changed to be an array of transactions.

Also in the future we will replace all of this with hashes.

Reply:
https://github.com/bpfkorea/agora/pull/603 has been merged, so this should be possible to do now.

Reply:
Do you think we should use `Set!Transaction` as a consensus data type?
Or would you rather write `Transacntion[]`?

Of course, we'll go to the hash later, but now.

Reply:
This issue is resolved by PR #609.
And I added a unit test case.